[32mINFO    [0m filename:test_1.py:108 退出成功